src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -128,7 +128,7 @@ public void addPbaAccountToOrganisation(Set<String> paymentAccounts, Organisatio
         }
     }
 
-    private void addSuperUserToOrganisation(
+    public void addSuperUserToOrganisation(
```

Comment:
These methods were made public as per discussion and agreement with @shashank-rastogi:

> a new instance of Organisation is created in the method and then passed to saveOrganisation method, so I am unable to do a verify on this new Organistion instance as I cannot mock it